src/Main.java

```diff
@@ -0,0 +1,96 @@
+
+import com.sun.source.tree.IfTree;
+
+import java.io.IOException;
+import java.util.*;
+
+public class Main {
+
+    static final String INPUT_FILEPATH = "testfiles/testIn.csv";
+    static final String OUTPUT_FILEPATH = "outputFile.csv";
+
+    public static void main(String[] args) {
+
+        //Read in the .csv file
+        List<Map<String, String>> rowsOfData = CsvReader.readCsv(INPUT_FILEPATH);
```

Comment:
We'd want the file to be inputted by the user. We'd make this a command line java app and we'd have as inputs the input file path and the output file path